database/handlers/project.go

```diff
@@ -185,3 +185,45 @@ func (client Client) GetProjectsByOrganization(id string) []database.ProjectThum
 	}
 	return projects
 }
+
+/*
+ * The Following function gets the count of projects per year for a given organization.
+ * 
+ * Args: id (id of organization)
+ * Returns : counts (slice contatining count of projects and years in a count object)
+ */
+
+ func (client Client) GetCountOfProjectsByParentOrgID(id string) []database.ProjectCountByYear {
+	queryStmt :=
+		`
+        SELECT programYear, COUNT(*) as count
+		FROM projects WHERE organizationId = $1
+		GROUP BY programYear
+		ORDER BY programYear;
```

Comment:
Indent the query uniformly.
Also, `COUNT(*)` is usually terrible for performance. Instead, always use a non-null attribute (like primary key) in counting. `COUNT(id)` is a good candidate here.

Reply:
Good to know. I'll change it and request a review once done